src/tasks/tasks.py

```diff
@@ -359,9 +360,9 @@ def process_split(self, split, indexers) -> Iterable[Type[Instance]]:
         ''' Process split text into a list of AllenNLP Instances. '''
         return process_single_pair_task_split(split, indexers, is_pair=True,
                                               classification=False)
-    def update_metrics():
-        # currently don't support metrics for regression task
-        # TODO(Yada): support them! 
+    def update_metrics(self, logits, labels, tagmask=None):
+        self.scorer1(mean_squared_error(logits, labels))
```

Comment:
Add a comment here that this is average MSE (which is why you add a MSE here)

Reply:
Done. Actually there is also similar comment in the __init__ of this task.